### ISSUE TITLE
Line numbers in high contrast theme

### ISSUE THREAD

**Issue by patx** [elementary/code#548]
The line numbers have the same white background as the rest of my text (white). It makes it very confusing to look at code that is not indented. The two other themes do not have this issue as they both have a grey pellucid background behind the lines numbers. I think the same should be done for the high contrast theme. I like my white background but you need separation between the line numbers and the text.

**Comment by patx** [comment]
@jeremypw Yes that would be good. I see in the code that the other two themes have their own stylesheets but the high contrast theme acts differently. See [here](https://github.com/elementary/code/blob/55ec80890e1d6b773efe073067890abf5b774581/src/Widgets/DocumentView.vala#L200). I think we should have a stylesheet for the high contrast theme as well.

Thoughts on this @danirabbit ?

**Comment by jeremypw** [comment]
@patx I have experimented with a modified elementary-light theme to make it higher contrast (see #1602). Here is how it looks so far:

![Image](https://github.com/user-attachments/assets/1c2dfb28-a338-4808-8991-8882b67f9a52)

**Comment by patx** [comment]
@jeremypw this looks pretty good. I think a grey background for the numbers looks more modern with the white main background. I also assume elemenatry wants to keep their current light theme as-is. We need to create a new XML style sheet for the high contrast/white theme.
